src/SDF.cc

```diff
@@ -97,6 +97,17 @@ std::string findFile(const std::string &_filename, bool _searchLocalPath,
     filename = filename.substr(idx + sep.length());
   }
 
+  // First check the local path, if the flag is set.
+  if (_searchLocalPath)
+  {
+    std::string path = sdf::filesystem::append(sdf::filesystem::current_path(),
+                                                filename);
```

Comment:
```suggestion
    std::string path = sdf::filesystem::append(sdf::filesystem::current_path(),
                                               filename);
```

Reply:
@ahcorde I have made the suggested changes in the latest commit